sklearn/tree/_tree.pyx

```diff
@@ -91,22 +91,29 @@ cdef class TreeBuilder:
         """Build a decision tree from the training set (X, y)."""
         pass
 
-    cdef inline _check_input(self, object X, cnp.ndarray y,
-                             cnp.ndarray sample_weight):
+    cdef inline _check_input(
+        self,
+        object X,
+        cnp.ndarray y,
+        cnp.ndarray sample_weight
+    ):
         """Check input dtype, layout and format"""
         if issparse(X):
+            # csc is fortran by default
```

Comment:
I see where are you coming from, but I think it could be confusing to say a csc matrix is fortran, given that the sparse matrix is represented by three 1D arrays. 

Is "fortran" commonly used with sparse matrices?

Reply:
I revised the comment, but let's see what benchmarks say